packages/sql/src/CopyFromManager.ts

```diff
@@ -0,0 +1,28 @@
+import { Credentials } from './credentials';
+import RequestManager from './RequestManager';
+
+export class CopyFromManager extends RequestManager {
+
+  constructor(credentials: Credentials) {
+    super({ ...credentials, server: credentials.server + 'api/v2/sql/copyfrom' });
+  }
+
+  public query(q: string) {
```

Comment:
It looks like `query` has the same implementation here and in `CopyToManager` so it could be moved up to RequestManager

Reply:
Oh this is a pointless method, I'll remove it